implementations/go-jsonschema/bowtie_jsonschema.go

```diff
@@ -166,3 +136,34 @@ func main() {
 		log.Println(err)
 	}
 }
+
+func jsonschemaVersion() string {
+	buildInfo, ok := debug.ReadBuildInfo()
+	if !ok {
+		panic("Failed to read build info")
+	}
+
+	var version = ""
+	for _, dep := range buildInfo.Deps {
+		if strings.Contains(dep.Path, "github.com/santhosh-tekuri/jsonschema") {
+			version = dep.Version
+			break
+		}
+	}
+	return version
+}
+
+func jsonReader(v interface{}) io.ReadCloser {
+	b, err := json.Marshal(v)
```

Comment:
Interesting! Your implementation indeed doesn't have a way to pass in a thing that has already been deserialized? I thought I must have just been missing it.

Reply:
such cases are rare so I did not bother implementing it

Reply:
also to support big numbers I unmarshal schema with `json.UseNumber()`

Reply:
Got it! Makes sense (certainly on the latter) -- which just to confirm, does that mean that the roundtrip here was lossy (because we already have deserialized the input without calling `.UseNumber()`?

Reply:
might be lossy in case of exact float representation. seems some optional tests with multipleOf might fail if floating precision is lost. let me fix this also...

Reply:
Cool (those aren't running yet, doing so is #36, which is why they may not appear yet as failures).

Reply:
done. but I noticed in website there are 3 test errors but when locally ran it has 14 errors.

not so easy to find which are failing

Reply:
Help welcome! As I say, it should be simple to improve the report for filtering and sorting, it's what's being worked on now.

Reply:
can you run `Test Suite Report` action once for me

Reply:
Done! Should be up to date, though I'm making more minor tweaks to the report on the way to the above so I'll be rebuilding it again in ~5 mins.

I'm also happy to add you as a contributor if you'd like (and then you can run it as often as you'd like yourself as well)

Reply:
Hooray! Awesome, glad to hear.